src/site-layout/SiteLayout.js

```diff
@@ -2,13 +2,18 @@ import PropTypes from "prop-types";
 import React from "react";
 import styled from "styled-components";
 import BrandingBar from "../branding-bar";
+import { CONTAINER_WIDTH } from "../constants";
+import Footer from "../footer";
 import NavBar from "../nav-bar";
 import PageRoutes from "../page-routes";
 
-const CONTAINER_WIDTH = 1144;
 const NAV_WIDTH = 150;
 
 const SiteContainer = styled.div`
+  width: 100%;
+`;
+
+const BodyWrapper = styled.div`
```

Comment:
The footer needs to extend the width of the screen so I had to push `SiteContainer` up a level since it was previously confined to a width of `1144px`